db/migrate/20151103223107_drop_properties.rb

```diff
@@ -0,0 +1,5 @@
+class DropProperties < ActiveRecord::Migration
+  def change
```

Comment:
should we make it reversible, then? I can't think of reasons why we would want to rollback, but I know it's always a good idea to make migrations reversible...